src/PrestaShopBundle/Resources/config/routing/admin/improve/international/state.yml

```diff
@@ -1,6 +1,36 @@
 admin_country_states:
   path: /
   methods: [GET]
+  #condition needed to differentiate from admin_states_index path
+  condition: "request.isXmlHttpRequest()"
   defaults:
     _controller: 'PrestaShopBundle:Admin\Improve\International\State:getStates'
     _legacy_controller: AdminStates
+
+admin_states_index:
+  path: /
```

Comment:
Let's avoid same URLs 😉 it is better according to KISS principle and avoids complex debugging issues

Reply:
changed to  path: /list